game.py

```diff
@@ -0,0 +1,54 @@
+from calculation.summation import Summation
+from calculation.calculation_class import Calculation
+
+
+class Game:
+    @staticmethod
+    def run_game():
+        summation = Summation(1, 10)
+
+        username = input("Здравей! Как се казваш?\n" "Моето име е: ")
+        print(f"Добре да се забавляваме, {username}. Нека да започнем с действие събиране!")
+        sum_not_correct_answer = 0
+        sum_correct_answer = 0
```

Comment:
Better names names for the variables could be: `incorrect_count` and `correct_count`